.eslintrc.json

```diff
@@ -1,27 +1,32 @@
 {
-    "root": true,
-    "env": {
-        "browser": true,
-        "es2021": true
+  "root": true,
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:@typescript-eslint/recommended-requiring-type-checking",
+    "plugin:jest/recommended",
+    "plugin:jest/style",
+    "plugin:prettier/recommended",
+    "plugin:react/recommended"
+  ],
+  "parser": "@typescript-eslint/parser",
```

Comment:
It allows us to do the deeper `plugin:@typescript-eslint/recommended-requiring-type-checking`.

We should look at adding it in oEQ, however there is caution advised for adding it to larger code bases as it can slow things down. We'd just have to try and see how it goes.